src/docs/unbricking/index.md

```diff
@@ -33,6 +33,7 @@ While this is somewhat device-specific, the main points are the same:
 
 * Disconnect all external power
 * Remove bottom cover (screws are often located under rubber feet or strips)
+        - In some Chromebooks, you eill need to remove the keyboard, and as mentioned in [Disabling write protect via Battery](/firmware/battery.html), theres a ribbon wire thats needs to be CAREFULLY removed.
```

Comment:
"Will" is spelled wrong. 

This sentence should be reworded. Current wording makes it sound like all Chromebooks have this cable.